plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -165,8 +183,9 @@ def main():
     argument_spec.update(dict(
         api_token=dict(type='str', required=True, no_log=True),
         gitlab_group=dict(type='str', required=True),
-        gitlab_user=dict(type='str', required=True),
-        state=dict(type='str', default='present', choices=['present', 'absent']),
+        gitlab_user=dict(type='str', required=False),
```

Comment:
If GitLab usernames cannot contain commas, it would be better to convert `gitlab_user` to a list of strings. (If you provide a string to a list, Ansible splits the string by commas to get a list.)

Reply:
So it would be compatible with the current implementation if we only have gitlab_user as a list, right?

Reply:
Yes.